.changeset/soft-bananas-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vercel/build-utils": patch
```

Comment:
Now that we're changing the call signature of `getEnvForPackageManager` this is probably more than a patch change, arguably major.

Reply:
Yeah, I shouldn't mess with this. Will revert.

Reply:
https://github.com/vercel/vercel/pull/11596/commits/cfe6b1ad44c54c072af40a4833235a722e906ebe